book/invalidation.md

```diff
@@ -758,8 +758,7 @@ very specific sequence of changes.
 [under-invalidation]: https://developer.chrome.com/articles/layoutng/#under-invalidation
 [hard-to-find]: https://developer.chrome.com/articles/layoutng/#correctness
 
-
-Recursive invalidation
+Recursive Invalidation
```

Comment:
I'm not doing this in my PRs (I'll fix it in post, as in, when I copy over changes to the copyeditor's version) but it's worth doing so let's keep it in.